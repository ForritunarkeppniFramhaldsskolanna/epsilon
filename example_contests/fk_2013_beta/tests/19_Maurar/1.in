2
10 3
2 6 7
214 7
11 12 7 13 176 23 191
