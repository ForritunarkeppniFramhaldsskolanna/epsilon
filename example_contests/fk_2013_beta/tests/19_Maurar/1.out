4 8
38 207
